src/annotator/features.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const events = require('../shared/bridge-events');
+
+let _features = {};
+
+module.exports = {
+
+  init: function(crossframe) {
+    crossframe.on(events.FEATURE_FLAGS_UPDATED, this.set);
+  },
+
+  set: function(features) {
```

Comment:
I don't think `set()` is part of `features.js`'s public API anymore, as long as the tests can call it indirectly by sending crossframe events then it only needs to be called directly by internal `features.js` code?